fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomeManager.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.EDGE_MAP;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.HILLS_MAP;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.INJECTED_BIOME_LIST;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.RIVER_MAP;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.SHORE_MAP;
+
+import net.fabricmc.fabric.impl.biomes.BiomeAssociate;
+import net.fabricmc.fabric.impl.biomes.BiomeLists;
+import net.minecraft.util.Pair;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.biome.Biome;
+
+public class OverworldBiomeManager
```

Comment:
Also this class could do with a better name. It does not actually manage biomes, it's more of an injector.

Reply:
How about BiomeUtils and OverworldBiomeUtils

Reply:
`BiomeAssoiciates`

Reply:
That seems weird, and not user friendly

Reply:
I'll keep them as BiomeUtils and OverworldBiomeUtils for now since I made a non-injection method in OverworldBiomeUtils

Reply:
I could rename BiomeUtils to CommonBiomeUtils?